src/components/WorkHeroSection/WorkHeroSection.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Box, Grid } from 'grommet';
+
+import { Heading } from '../../legos/typography/Heading';
+import { StyledButton } from '../../legos/Button/Button';
+import { Text } from '../../legos/typography/Text';
+
+const skillItems = [
+  'Branding',
+  'Visual Design',
+  'UI & interactions',
+  'Frontend',
+  'Backend',
+  'Mobile apps',
+];
+export const WorkHeroSection = () => (
+  <Box
+    height="695px"
+    background={{
+      size: 'small',
+      position: 'right',
+      image: 'url(./assets/background-work.svg)',
+    }}
+  >
+    <Heading alignSelf="center" level={1} color="#104065" wordBreak="normal">
+      Work
+    </Heading>
+    <Grid columns={{ count: 2, size: 'auto' }} gap="small" fill="horizontal">
+      <Box
+        justify="center"
+        align="start"
+        pad={{ bottom: 'small', left: 'large' }}
+      >
+        <Box pad={{ bottom: 'medium' }}>
+          <Heading level={2} color="#104065" wordBreak="normal">
```

Comment:
Consider passing "brand" as the color:
```suggestion
          <Heading level={2} color="brand" wordBreak="normal">
```